cv32e40p/sim/core/Makefile

```diff
@@ -421,21 +412,33 @@ xrun-all: $(CV_CORE_PKG) $(RTLSRC_PKG) $(RTLSRC) $(TBSRC_PKG) $(TBSRC)
 		-f $(CV_CORE_MANIFEST) \
 		$(TBSRC_PKG) $(TBSRC)
 
-.PHONY: xrun-hello-world
-xrun-hello-world: xrun-all $(VERI_CUSTOM)/hello-world/hello-world.hex
-	$(XRUN) \
-		$(XRUN_FLAGS) \
-		-f $(CV_CORE_MANIFEST) \
-		$(TBSRC_PKG) $(TBSRC) \
-		+firmware=$(VERI_CUSTOM)/hello-world/hello-world.hex
-
-.PHONY: xrun-custom
-xrun-custom: xrun-all $(VERI_CUSTOM)/$(CUSTOM_PROG)/$(CUSTOM_PROG).hex
+#.PHONY: xrun-hello-world
```

Comment:
Can we just get rid of these targets instead of commenting?

Reply:
Yes we can.  Good catch.